ci-operator/config/stolostron/policy-collection/stolostron-policy-collection-main__opp-ocp4.12-interop.yaml

```diff
@@ -55,6 +64,8 @@ tests:
     - ref: acm-policies-openshift-plus-setup
     - ref: acm-policies-openshift-plus
     - chain: cucushift-installer-check-cluster-health
+    - ref: acm-tests-clc-create
+    - ref: acm-fetch-managed-clusters
```

Comment:
Does this just validate the managed clusters are setup?  Could we make it the last test in the `pre` list to make sure OPP validations do not impact managed cluster availability?  I'm not sure if this needs to be immediately after the CLC setup or even if may it could run twice -- after CLC and then as the last test.

Reply:
acm-fetch-managed-clusters queries the managed clusters, generates their credentials and stores them.
Those credentials will be used as input parameters for other components such as ALC, GRC and OBS